### PR TITLE
Length, silence and volume check

### PR DESCRIPTION
**Pull request by YuhanYue**

### CODE REVIEW

File: server/app/config.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "vad_mode":3,
+    "silence_ratio":0.9,
+    "lowest_dBFS":-18,
+    "shortest_length_for_sentence":0.15,
```
Comment (ryangawei) [inline server/app/config.json:5]: Why is the shortest length 0.15? I think Aanchan suggested with 0.3s

File: server/app/__init__.py [hunk 2]
```diff
@@ -11,6 +11,7 @@
 
 def create_app():
     app = Flask(__name__)
+    app.secret_key = b'_5#y2L"F4Q8z\n\xec]/'
```
[the remark this answers is not in the document]
Comment (YuhanYue) [reply server/app/__init__.py:14]: moved "app_secrect_key" field into config file

File: server/app/config.json (hunk 1 above)
Comment (YuhanYue) [reply server/app/config.json:5]: fixed

File: server/app/scripts/silence_check.py [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+    if triggered:
+        sys.stdout.write('-(%s)' % (frame.timestamp + frame.duration))
+    sys.stdout.write('\n')
+    # If we have any leftover voiced audio when we run out of input,
+    # yield it.
+    if voiced_frames:
+        yield b''.join([f.bytes for f in voiced_frames])
+
+
+def is_valid_speech(vad, f):
```
Comment (ryangawei) [inline server/app/scripts/silence_check.py:146]: Can you rename it to `get_silence_ratio`?

**Review by ryangawei — CHANGES_REQUESTED** [review]

**Review by ryangawei — APPROVED** [review]

**Pull request merged**